monitor/flowgen/bridging/bridging.go

```diff
@@ -111,3 +125,39 @@ func Jobs(networkID netconf.ID, backends ethbackend.Backends, owner common.Addre
 
 	return []types.Job{job1, job2}, nil
 }
+
+// estimateOrderSize checks the current balance of the flowgen EOA and returns
+// the maximal possible order size or nil if the minimal balance is reached.
+func estimateOrderSize(
+	ctx context.Context,
+	networkID netconf.ID,
+	client ethclient.Client,
+	owner common.Address,
+	conf flowConfig,
+) (*big.Int, error) {
+	balance, err := client.BalanceAt(ctx, owner, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	thresholds, ok := eoa.GetFundThresholds(tokens.ETH, networkID, eoa.RoleFlowgen)
```

Comment:
you are hardcoding `ETH` token, but the `openOrder` logic supports dynamic native tokens, you call `app.AllTokens().Find(`. I'd suggest being more consistent, and either use hardcoded values or dynamic native chain tokens, not both. Suggest passing in `srcToken.Token` rather.